examples/adaptivity/adaptivity_ex3/adaptivity_ex3.C

```diff
@@ -555,9 +567,10 @@ int main(int argc, char ** argv)
 #ifdef LIBMESH_HAVE_EXODUS_API
   // Write out the solution
   // After solving the system write the solution
-  // to a ExodusII-formatted plot file.
-  ExodusII_IO (mesh).write_equation_systems ("lshaped.e",
-                                             equation_systems);
+  // to a GMVIO-formatted plot file (p levels can be visualized).
+  GMVIO gmv_io (mesh);
+  gmv_io.p_levels();
+  gmv_io.write_equation_systems ("lshaped.gmv", equation_systems);
```

Comment:
This code doesn't make sense to have inside a `LIBMESH_HAVE_EXODUS_API` ifdef. Aside from that, we don't really use the GMV binary file format much at all nowadays, so it would be better to continue writing Exodus files from this example, and plot the p-levels using an `ExplicitSystem`... there are most likely examples of this somewhere we could dig up if necessary.